.gitignore

```diff
@@ -21,3 +21,4 @@ charts/ti-key-release-2/requirements.lock
 
 # remove mac files
 *.DS_Store
+charts/tsi-node-setup-v1.6.0.tgz
```

Comment:
How about `charts/*.tgz`?

Reply:
Since ti-key-release-1 is already included inside ti-key-release-2, we only want to ignore the ti-key-relase-1. Fixed.